modules/settings/Settings.js

```diff
@@ -13,8 +13,15 @@ function getLocalStorage() {
 
     // eslint-disable-next-line no-invalid-this
     const global = typeof window === 'undefined' ? this : window;
+    let storage;
 
-    return global.localStorage;
+    try {
+        storage = global.localStorage;
```

Comment:
I don't think this will catch the non-existence of localStorage across all browsers, specifically FireFox.